docs/pages/includes/database-access/auto-user-provisioning-common-teleport-role.mdx

```diff
@@ -0,0 +1,27 @@
+To specify the database roles a user should be assigned within the database,
+use the `db_roles` role option:
+
+```yaml
+kind: role
+version: v7
+metadata:
+  name: auto-db-users
+spec:
+  options:
+    # create_db_user enables automatic user provisioning for matching databases
+    create_db_user: true
+  allow:
+    db_labels:
+      "*": "*"
+    db_names:
+    - "*"
+    # db_roles is a list of roles the database user will be assigned
+    db_roles:
+    - reader
+    - "{{internal.db_roles}}"
+    - "{{external.db_roles}}"
+```
+
+With automatic user provisioning, users always connect to the database with
+their Teleport username so the `db_users` role field is ignored for roles
+that have database user provisioning enabled.
```

Comment:
This was a big surprise to me when I first tried that feature. I think it is worth highlighting that automatic user creation takes away the choice of the username. It makes sense from the security and implementation point of view, but I think it is important.

Reply:
I converted the existing block in the connect section to an Admonition. How's that look
[b8a34b5](https://github.com/gravitational/teleport/pull/33084/commits/b8a34b56bb211f5aec2731047f787ad808c73a03)